### ISSUE TITLE
Saving image error in Bluesky

### ISSUE THREAD

**Issue by rozchh** [bluesky-social/social-app#7211]
### Steps to Reproduce

1. Go to Bluesky.
2. Click on any image
3. Press "Save" on Image
4. Image cannot be saved. It is read as "Failed to save image." And cause by "missing MEDIA_LIBRARY write permission"

### Attachments

![Screenshot_20241220-174322_Bluesky](https://github.com/user-attachments/assets/af7a3d05-aa16-4c7f-b2f4-a671355fb48f)

### What platform(s) does this occur on?

Android

### Device Info

Galaxy A12

### What version of the app are you using?

Build version: 1.96.1.539; Bundle info: ab0697a (prod); Bundle date: 24122000; Platform: android; Platform version: 31

### Additional Information

I don't know how to fix this media

**Comment by u83rmensch** [comment]
Im having the same issue. 

version 1.96.1.539 ab0697a (prod) 

tried to nuke the whole app. cleared the cache, storage, and uninstalled. rebooted the phone and re-installed from the play-store. still having the same issue.

**Comment by KiraCoding** [comment]
I have the same issue; tried reinstalling and manually force enabling all permissions, still doesn't work.

![Screenshot_20241221-174316_Bluesky.jpg](https://github.com/user-attachments/assets/3baacdf4-81fe-4f40-ad7f-424e71fa09fa)

## Device details
- `Galaxy A70`
- `Android 11`

## Build version
`Build version: 1.96.1.539; Bundle info: ab0697a (prod); Bundle date: 24122000; Platform: android; Platform version: 30`

**Comment by gaearon** [comment]
Yeah we’re aware of the issue and have a fix on the way to the store. Not sure how long it’ll take to get there.

**Comment by X1Vi** [comment]
> Yeah we’re aware of the issue and have a fix on the way to the store. Not sure how long it’ll take to get there.

Any related PRs that I can see or refer ? Or should I make one myself ?
